generator/build.yaml

```diff
@@ -1,6 +1,5 @@
 builders:
-  retrofit:
-    target: ":retrofit"
+  retrofit_generator:
```

Comment:
With the old name you just need to reference it like `retrofit_generator:retrofit`, then it would work. This is changing the name - so it could break some users if they are correctly targetting it today, fwiw. But this is also an easier name for people to  use (they can just say `retrofit_generator`).

Reply:
In line 2 `retrofit_generator:` replace to `retrofit_generator:retrofit`?